### ISSUE TITLE
Enable minimalistic config.yaml

### ISSUE THREAD

**Issue by yonkornilov** [yonkornilov/snowcapper#26]
```
packages:
  - name: test
    files:                                
      - path: /etc/vault/config.hcl
        mode: 0700
        content: |
          storage "file" {
            path    = "/mnt/vault/data"
          }

          listener "tcp" {
            address     = "0.0.0.0:8200"
            tls_disable = 1
          }

          disable_mlock = "true"
    inits:
      - type: command
        content: echo nothing is installed
```
Gives `Runtime error: Packages: (0: (Binaries: cannot be blank.).).`

In another example,

```
packages:
  - name: test
    inits:
      - type: command
        content: echo nothing is installed.
```

Gives `Runtime error: Packages: Either Files or Services must be present.`

Configurations should be as minimalistic as the user desires.

**Comment by yonkornilov** [comment]
This proves minimalistic configs are now possible https://github.com/yonkornilov/snowcapper/pull/27/commits/bda8e8950a65de94eb0042b00b04610967d074fb

**Comment by yonkornilov** [comment]
Closed after #27
